code/modules/client/preference_setup/occupation/skill_selection.dm

```diff
@@ -0,0 +1,194 @@
+/datum/preferences
```

Comment:
Skill selection is not available to players now?

Reply:
yes its disabled but i left it in case it will be re-implemented as stat selection